builtin/providers/rancher/provider.go

```diff
@@ -71,7 +73,7 @@ func init() {
 
 		"secret_key": "API secret used to authenticate with the rancher server",
 
-		"api_url": "The URL to the rancher API",
+		"api_url": "The URL to the rancher API, must include version uri (ie. v1 or v2-beta)",
```

Comment:
Discussed internally, we are going to go back to the original behavior and let the go-rancher client figure it out. The go-rancher client can take a v1, v2 or / url and do the right thing. I'll make it so the other client types in the provider build the URLs off of it.